src/Console/App.php

```diff
@@ -13,14 +13,14 @@
 use Codelicia\Depreday\UI\Message;
 use Codelicia\Depreday\UI\Phrases;
 use DateTimeImmutable;
-use Psl;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 use function array_map;
 use function is_array;
 use function is_string;
+use function Psl\invariant;
```

Comment:
Why did you import this?

Reply:
Codana is [complaining about invariant](https://github.com/codelicia/depreday/pull/22/files). 

Importing doesn't help so it's back to how it was.